### ISSUE TITLE
Hacktoberfest contribution: gatsby-source-etsy

### ISSUE THREAD

**Issue by dylanon** [gatsbyjs/gatsby#18291]
Contributing my source plugin as per #18129!

I've been working on `gatsby-source-etsy`, a source plugin that pulls featured listings from an Etsy store. The plugin downloads the featured listings and their associated images, which are then queryable via GraphQL. The images are compatible with `gatsby-image` too.

I'm looking to add some more configuration options soon and it probably needs a couple API changes before it hits 1.0, but for now [here's the published package](https://www.npmjs.com/package/gatsby-source-etsy).

**Comment by marcysutton** [comment]
This is AWESOME, @dylanon!! What a cool idea. As a future suggestion, adding a short description  in the README on how the plugin might be used (like what kind of a site someone might want to build with it) would help inspire adoption.

Since this issue was created, we've come up with more of a plan for your contribution to count toward Hacktoberfest. If you're able to do so, can you [create a Pull Request](https://www.gatsbyjs.org/contributing/how-to-open-a-pull-request/) against the announcement blog post to add a link to your plugin? It will give your project a bit more visibility, too! 

There's a good spot for it at the end of the Ecosystem section of the post, which talks about this new process (I've also updated the umbrella GitHub issue): https://github.com/gatsbyjs/gatsby/blob/master/docs/blog/2019-10-08-hacktoberfest-2019/index.md#ecosystem

Here's a suggestion on what to add, you can copy and paste it if you want:
```
- [gatsby-source-etsy](https://www.npmjs.com/package/gatsby-source-etsy)
```

Thanks so much, and let me know if you have any questions!

**Comment by marcysutton** [comment]
Closed with #18482. thanks again!
